apps/ui/src/composables/useActions.ts

```diff
@@ -525,19 +526,21 @@ export function useActions() {
   async function delegate(
     space: Space,
     networkId: NetworkID,
+    delegationType: DelegationType,
     delegatee: string,
     delegationContract: string
   ) {
     if (!web3.value.account) return await forceLogin();
 
-    const network = getReadWriteNetwork(networkId);
+    const network = getReadWriteNetwork(networkId, true);
```

Comment:
Didn't think about it initially, but can we make second argument options object with flags inside? This way it will be cleaner when it's used:
```js
const network = getReadWriteNetwork(networkId, { allowDisabledNetwork: true });

// vs
const network = getReadWriteNetwork(networkId, true);